components/ContentSearch/index.js

```diff
@@ -88,6 +88,10 @@ const ContentSearch = ({ onSelectItem, placeholder, label, contentTypes, mode, e
 
 				setSearchResults(newResults);
 				setIsLoading(false);
+			}).catch( (error) => {
```

Comment:
@mitogh What do you think about giving the component a new piece of state, like:

```JSX
const [error, setError] = useState();
```

And then using the error from `catch` to display a message below the text area. The message could disappear once the user starts typing again. 

Using a notice, as you mentioned, might be a good idea, but this component is meant to be useful outside the editor context (e.g., on other admin screens), so notices might not always be available.

Reply:
*Optional*  A little outside the scope of the PR, but what do you think about making this callback `async` and handling this error with `try`/`await`/`catch`/ instead of `then`/`catch`?

Reply:
I think having a state to manage the errors is the way to go I can polish this a bit more to make it happen and use await as well.

In terms of UI I can add the error message at the top of the component as default behavior but allowing other behavior if provided via a prop.

Thanks.

Reply:
Agreed an error message would be good. Also, this will probably need to be reconfigured based on some other changes in trunk now.

Reply:
Thanks let me take over the conflicts and UI errors.

Reply:
Here's a vide of how it looks like when being used: 

- https://share.vidyard.com/watch/a48t1f7tHKevRZRgCteuGq?

Reply:
Is this ready for re-review?